ReVampyr_work.py

```diff
@@ -274,4 +277,210 @@
     GJmK_11_r, GJmK_11_i = spinorb1.dot(GJmK_phi1)
 
     print('GJmK_11_r', GJmK_11_r)
-    print('E_C_G', E_tot_JK - GJmK_11_r - (2.0 *light_speed**2))
+    #print('E_C_G', E_tot_JK - GJmK_11_r - (2.0 *light_speed**2))
+
+#############################END GAUNT & START WITH GAUGE###################################
+if args.coulgau == 'gauge': 
+    spinorb2 = spinorb1.ktrs()
+    
+    #Definition of alpha vectors for each orbital
+    alpha1_0 =  spinorb1.alpha(0)
+    alpha1_1 =  spinorb1.alpha(1)
+    alpha1_2 =  spinorb1.alpha(2)
+
+    alpha2_0 =  spinorb2.alpha(0)
+    alpha2_1 =  spinorb2.alpha(1)
+    alpha2_2 =  spinorb2.alpha(2)
+
+    #Defintion of orbital * alpha(orbital)
+    cphi2_alpha1_0 = spinorb2.overlap_density(alpha1_0, prec)
+    cphi2_alpha1_1 = spinorb2.overlap_density(alpha1_1, prec)
+    cphi2_alpha1_2 = spinorb2.overlap_density(alpha1_2, prec)
+   
+    cphi2_alpha2_0 = spinorb2.overlap_density(alpha2_0, prec)
+    cphi2_alpha2_1 = spinorb2.overlap_density(alpha2_1, prec)
+    cphi2_alpha2_2 = spinorb2.overlap_density(alpha2_2, prec)    
+    
+    #Definitioin of Gauge operator 
+    O = r3m.GaugeOperator(mra, 0.0001, 60.0, prec)
+    print('Gauge operator DONE')
+
+    #Definition of Gaunt two electron operators       
```

Comment:
Gauge?

Reply:
Yes it is our operator where we gave r_min and r_max in the input file.